### DISCUSSION TITLE
Make valid GraphQL error reply when Keystone backend is initializing

### DISCUSSION THREAD

**Discussion by MurzNN** [keystonejs/keystone#5923] [Feature requests]
When the Keystone backend is offline (server is stopped), Admin UI shows the error:
```
NetworkError when attempting to fetch resource.
``` 
That is ok, but when Keystone is initializing (that can spend long time), Admin UI shows the error:
```
JSON.parse: unexpected character at line 1 column 1 of the JSON data
```
because on `/api/graphql` it receives the HTML with "Keystone is loading..." instead of JSON-formatted reply.

Maybe on initializing stage we can replace the HTML reply on GraphQL endpoint to some valid GraphQL reply with the same error text "Keystone is loading..."?

**Comment by bladey** [comment]
Thank you for the suggestion @MurzNN.
